app/src/main/java/de/westnordost/streetcomplete/user/QuestStatisticsByQuestTypeFragment.kt

```diff
@@ -62,11 +63,9 @@ class QuestStatisticsByQuestTypeFragment : Fragment(R.layout.fragment_quest_stat
         val clickableContainer = FrameLayout(ctx)
         clickableContainer.layoutParams = ViewGroup.LayoutParams(256,256)
         // foreground attribute only exists on FrameLayout up until KITKAT
```

Comment:
missed to remove comment. Suggest to Ctrl+F for Kitkat, Ice Cream Sandwich and so on

Reply:
Thanks!